layers/sflow.go

```diff
@@ -696,8 +696,8 @@ const (
 	SFlowTypeLACPCounters               SFlowCounterRecordType = 7
 	SFlowTypeProcessorCounters          SFlowCounterRecordType = 1001
 	SFlowTypeOpenflowPortCounters       SFlowCounterRecordType = 1004
-	SFlowTypePORTNAMECounters           SFlowCounterRecordType = 1005
```

Comment:
I agree, that these names don't follow the conventions correctly, we can't change them now, since this might break existing implementations.

Reply:
Agreed, I suspected this could be an issue :) no problem to revert it